mhs-reference-implementation/Pipfile

```diff
@@ -16,10 +17,10 @@ ldap3 = "*"
 python_version = "3.7"
 
 [scripts]
-unittests = 'python -m unittest -v'
-unittests-cov = 'coverage run -m unittest -v'
+unittests = 'python -m xmlrunner -o test-reports -v'
+unittests-cov = 'coverage run -m xmlrunner -o test-reports -v'
 coverage-report = 'coverage report'
 coverage-report-xml = 'coverage xml'
 coverage-report-html = 'coverage html'
-inttests = 'python -m unittest discover -p "int_*" -v'
+inttests = 'python -m xmlrunner discover -o test-reports -p "int_*" -v'
```

Comment:
Guessing these scripts will want modifying in the `common` module as well depending on which PR goes in first